eng/pipelines/corefx-base.yml

```diff
@@ -138,7 +138,7 @@ jobs:
                 nugetConfigPath: 'NuGet.config'
                 restoreDirectory: '$(Build.SourcesDirectory)\.packages'
               env:
-                VSS_NUGET_EXTERNAL_FEED_ENDPOINTS: '{"endpointCredentials": [{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/8f470c7e-ac49-4afe-a6ee-cf784e438b93/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"}]}'
+                VSS_NUGET_EXTERNAL_FEED_ENDPOINTS: '{"endpointCredentials": [{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/8f470c7e-ac49-4afe-a6ee-cf784e438b93/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"},{"endpoint":"https://devdiv.pkgs.visualstudio.com/_packaging/VS/nuget/v3/index.json", "username":"dn-bot","password":"$(dn-bot-devdiv-nuget-feed-read)"}]}'
```

Comment:
nit: can this be split on new lines?

Reply:
To be honest I don't know. @safern?

Reply:
Technically yes. I haven't tried for quoted stuff, but it should work due to the yaml syntax model. I would give it a try.